static/js/load_map.js

```diff
@@ -75,4 +76,60 @@ function initialize() {
             markers.push(nextMarker);
         });
     });
+
+    const stores = document.getElementById('Stores');
+    function getLocation() {
+        if (navigator.geolocation) {
+            return navigator.geolocation.getCurrentPosition(showPosition);
+        } else {
+            x.innerHTML = "Geolocation is not supported by this browser.";
+        }
+    }
+
+    function showPosition(position) {
+        x.innerHTML = "Latitude: " + position.coords.latitude +
+        "<br>Longitude: " + position.coords.longitude;
```

Comment:
The indentation here is weird. Also, should you really be setting `innerHTML` by hand?